content/en/docs/community_roles/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Community Roles
+weight: 10
+hide_readingtime: true
+---
+
+Individuals in following roles are the backbone of this community.
```

Comment:
in **the** following ..?

Reply:
The page loads a list of subpages...

<img width="939" alt="Screenshot 2024-09-13 at 05 14 50" src="https://github.com/user-attachments/assets/aae19194-0152-4702-be9d-127902bfdeb7">

Reply:
I meant you are missing the word `the` in the first sentence @eddie-knight

Reply:
```suggestion
Individuals the in following roles are the backbone of this community.
```